### ISSUE TITLE
Issue : Telemetry and osd

### ISSUE THREAD

**Issue by Karan1rao** [rizacelik/STM32F411CEU6_INAV_Firmware#31]
In STM 32 f411ceu6, I am not able to measure voltage accurately, I have properly divided voltage to get voltage of around 5 volts, however when I connect my battery to the flight controller it shows some voltage which can be fixed by adjusting voltage scale, however, when another battery of same cell count with higher or lower voltage is connected inav does not show the correct voltage, furthermore, i guess the led strip function is not working in inav 7.0 firmware

**Comment by rizacelik** [comment]
You need to make your calculation as 3.3V. Calculations are given below. 5V does not give the actual value.

Did LED INAV 6.1.1 also work? If it works, I can look for INAV 7.0.

![image](https://github.com/rizacelik/STM32F411CEU6_INAV_Firmware/assets/19993109/7c61aa7c-c44a-40b4-a03c-70df8486bd8e)

**Comment by rizacelik** [comment]
NOTE: Voltage pin will be PB0 in STM32F411CEU6. The above diagram is for STM32F405 board.

**Comment by Karan1rao** [comment]
Yes the led works in 6.1.1 firmware. And I would like to ask one more question, is the buzzer enabled in inav 7.0 firmware?

**Comment by rizacelik** [comment]
buzzer not supported.

**Comment by Karan1rao** [comment]
Voltage divider for 3.3 volts worked for me, thanks.       
Led is also working 🙂🙂, thanks bro.

**Comment by rizacelik** [comment]
It's great that everything went well. I wish you good work. Enjoyable flights.

**Comment by Karan1rao** [comment]
Bro how do i connect a telemetry Tx and Rx pin to the board as one Tx and Rx pin is used by  GPS and one Rx pin is used as PPM input, furthermore, i need to connect an osd to the controller and spi port is used by MPU 9250, do i connect osd with the same spi pin as of mpu 9250?
